src/WordPress.php

```diff
@@ -845,10 +882,26 @@ private function create_site( $assoc_args ) {
 		}
 
 		if ( ! empty( $this->cache_type ) ) {
+
 			$this->enable_object_cache();
 			$this->enable_page_cache();
+
+			if ( $this->is_vip ) {
+				EE::warning( 'Nginx-helper and wp-redis plugin is installed to enable cache. Please add it in your .gitignore to avoid it from git diff and commit' );
```

Comment:
Should we provide path or simple echo command to add it in `.gitignore`? @rahulsprajapati

Reply:
I think that is not necessary and developers can see the paths in git diff anyway.